### ISSUE TITLE
papermc() is not recognized in repository DSL block

### ISSUE THREAD

**Issue by Ungolim** [EndlessCodeGroup/BukkitGradle#45]
https://github.com/EndlessCodeGroup/BukkitGradle/blob/711bd72fe9a04564d66ab747ddfa4dcfbb726ba9/src/main/groovy/ru/endlesscode/bukkitgradle/Dependencies.groovy#L32

Judging by the above LoCs I would think that we can change the repository DSL block to something other than what is outlined in the README:
```groovy
repositories {
    spigot() // Adds spigot repo
}
```

(e.g.) to:
```groovy
repositories {
    papermc() // Adds Paper repo
}
```

However this results in the error: 
`> Could not find method papermc() for arguments [] on repository container of type org.gradle.api.internal.artifacts.dsl.DefaultRepositoryHandler.`

Am I wrong to assume this is the case?

**Comment by osipxd** [comment]
You're doing it right. What Gradle DSL you use, Kotlin or Groovy?

**Comment by Ungolim** [comment]
Groovy

Il giorno dom 29 nov 2020 alle 22:01 Osip Fatkullin <
notifications@github.com> ha scritto:

**Comment by osipxd** [comment]
Is your project published on GitHub? I could check it

**Comment by Ungolim** [comment]
> Is your project published on GitHub? I could check it

https://github.com/lepidotteri/ARM-Paper-Gradle/blob/master/build.gradle

I put 1.16.4 in there because that's the latest PaperMC (and vanilla) stable server version, but it's not yet supported by your plugin it seems

**Comment by osipxd** [comment]
Thank you! I'll be able to check it at the end of the week.

**Comment by osipxd** [comment]
Will be fixed in 0.9
